typings/cldrjs/cldrjs.d.ts

```diff
@@ -1,4 +1,112 @@
 declare module 'cldrjs' {
-	let cldrjs: any;
-	export = cldrjs;
+	import Cldr = require('cldr');
+	export = Cldr;
+}
+
+declare module 'cldr' {
+	import CldrEvent = require('cldr/event');
+
+	class Cldr {
+		/**
+		 * @requires cldr/unresolved
+		 */
+		protected static _raw: Object;
+
+		static localeSep: string;
+
+		protected _alwaysArray<T>(arg: T | T[]): T[];
+		protected _coreLoad(Cldr: Cldr, source: Object, cldrData: Object): Object;
+		protected _createError(code: string, attributes: Object): Error;
+		protected _itemGetResolved(Cldr: Cldr, path: string | string[], attributes: Object): Object;
+		protected _jsonMerge(...objects: Object[]): Object;
+		protected _pathNormalize(path: string | string[], attributes: Object): string[];
+		protected _resourceGet(data: Object, path: string): string | Object;
+		protected _validatePresence(value: any, name: string): void;
+		protected _validateType(value: any, name: string, check: Boolean, expected: string): void;
+		protected _validateTypePath(value: any, name: string): void;
+		protected _validateTypePlainObject(value: any, name: string): void;
```

Comment:
Hmmm...even with subclassing, I'm not sure if these should be here. I mean, someone extending a class can get at the parameters even without typing information. There are very few uses of "protected" in DefinitelyTyped; it would be worth looking more into how other developers handle this situation.